.github/workflows/on_push_pr.yaml

```diff
@@ -15,7 +15,6 @@ jobs:
 
   test-integration-nix-cgroups-v1:
     name: Run integration tests on *Nix
-    needs: [push-pr]
     runs-on: ubuntu-20.04
```

Comment:
It might be obvious and I think the renovate rules already ignore this, but we could we add a comment saying that requires an OS with cgroupsv1? (and we should not be updated). Just in case...

Reply:
there is a [step](https://github.com/newrelic/nri-docker/blob/e0f92c96aca758aca5e81abef616aa14c8eb83b9/.github/workflows/on_push_pr.yaml#L23) that check it :)